src/runtime/runtime_avr.go

```diff
@@ -37,7 +37,7 @@ func initUART() {
 	*avr.UBRR0H = 0
 	*avr.UBRR0L = 8
 	*avr.UCSR0B = avr.UCSR0B_RXEN0 | avr.UCSR0B_TXEN0 // enable RX and TX
-	*avr.UCSR0C = avr.UCSR0C_UCSZ0                    // 8-bits data
+	*avr.UCSR0C = avr.UCSR0C_UCSZ00                   // 8-bits data
```

Comment:
This breaks UART: I had to change it to `*avr.UCSR0C = avr.UCSR0C_UCSZ01 | avr.UCSR0C_UCSZ00` to get UART to work again.

Reply:
OK will add a commit to correct this.

Reply:
Done.